nginx/check-nginx-status.yaml

```diff
@@ -0,0 +1,132 @@
+---
+# Description
+#   Check NGINX status
```

Comment:
Should we mention here or in the instructions that the NGINX "stub status" module is required for this plugin to work; and a reference to the NGINX [docs](https://nginx.org/libxslt/en/docs/http/ngx_http_stub_status_module.html) or [blog](https://www.nginx.com/blog/monitoring-nginx/)??? 

Should we surface that information here or in the instructions section?